lib/aws_command.rb

```diff
@@ -89,6 +111,13 @@ def config()
           abort 'Error: you need to specify either --name or (--type and --env)'
         end
 
+        # Check if we install a custom openshift
+        if !ENV['OO_OPENSHIFT_BINARY'].nil?
+          ah.extra_vars['oo_openshift_binary'] = ENV['OO_OPENSHIFT_BINARY']
+          FileUtils.cp(ENV['OO_OPENSHIFT_BINARY'], 'roles/openshift_master/files')
+          FileUtils.cp(ENV['OO_OPENSHIFT_BINARY'], 'roles/openshift_minion/files')
+        end
```

Comment:
same as above, we can do this all in the playbook. We should always try to do as much inside of the playbooks as possible. Only do things in the wrapper if you absolutely can't do it in ansible directly.